src/js/components/Button/StyledButton.js

```diff
@@ -11,18 +11,32 @@ import {
 } from '../../utils';
 import { defaultProps } from '../../default-props';
 
+const radiusStyle = props => {
+  if (props.size && props.theme.button.size) {
```

Comment:
I think we should check for the specific size here. If the caller passes in `xxsmall` as the size, this will cause an exception.

Reply:
Done, thanks.